packages/xchain-thorchain/src/client.ts

```diff
@@ -582,6 +582,7 @@ class Client implements ThorchainClient, XChainClient {
       })
       const unsignedStdTx = await this.buildDepositTx(msgNativeTx)
       const fee = unsignedStdTx.fee
+      fee.gas = '1000000' // fee.gas temporarily hardcoded since the value returned is too high, causing "insufficient funds" errors for users with enough RUNE
```

Comment:
Not sure if `1000000` is correct, we discussed `10000000000` internally

cc/ @cbarraford

Reply:
We have decided to set `gas = 'auto'`. Since we have seen it might still fail for first attempt, we try to send `deposit` tx up to three times.